src/content/contributing/style-guide/content-standartization/index.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Content standartization
+description: Guide to use consistent terminology in content writing
+lang: en
+---
+
+This style guide aims to standardize certain aspects of writing content to make the contribution process smoother.
+
+## Use American English {#american-english}
+
+For words that have multiple spellings, use American English over British English.
+
+**For example:**
+
+- "decentralized" over "decentralised"
+- "color" over "colour"
+- "analyze" over "analyse"
+
+## Terminology {#Terminology}
+
+
+### Ethereum {#ethereum}
+
+Ethereum is a proper noun and should always be capitalized.
+
+- "Ethereum" not "ethereum"
+
+### Ether {#ether}
+
+Ether is a common noun and should not be capitalized unless at the beginning of a sentence. ETH, on the other hand, is a currency abbreviation (and ticker symbol) and should always be capitalized.
```

Comment:
Should we have a preference for "ether" over "ETH" or vice-versa? Personally I slightly prefer to use "ether" as its the name not the ticker symbol, but appreciate if others disagree or we are happy to use both.

Reply:
Good question... we definitely tend to shorten to ETH more often than not. Personally fairly indifferent, but agree "ether" feels more pure, given it's not a ticker.

On a similar note... "ether" vs "Ether" (when not starting a sentence)... we went with "ether" originally because of the white paper, but worth noting that the Yellow Paper consistently uses capitalized "Ether" when referring to the currency unit.

If we want to address capitalization I'd suggest we address that separately, but we may want to consider doing a codebase find/replace for this in the future.